presto-main/src/main/java/com/facebook/presto/operator/SortedPositionLinks.java

```diff
@@ -152,13 +153,14 @@ public Factory build()
             return new Factory()
             {
                 @Override
-                public PositionLinks create(Optional<JoinFilterFunction> lessThanFunction)
+                public PositionLinks create(Optional<JoinFilterFunction> lessThanFunction, List<JoinFilterFunction> filterFunctions)
```

Comment:
I would rename `filterFunctions` to `inequalityFilterConjuncts` or `inequalityFilterExpressions` and then either remove `lessThanFunction`.
Or rename it to `combinedIneqalityFIlterExpressions` and ensure that it is clear when this is constructed that it is just a conjunction of function passed in the other parameter.